fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBinding.kt

```diff
@@ -0,0 +1,145 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations
+
+import android.content.Context
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.distinctUntilChangedBy
+import kotlinx.coroutines.flow.mapNotNull
+import mozilla.components.browser.state.selector.findTab
+import mozilla.components.browser.state.state.BrowserState
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.concept.engine.translate.initialFromLanguage
+import mozilla.components.concept.engine.translate.initialToLanguage
+import mozilla.components.lib.state.helpers.AbstractBinding
+import org.mozilla.fenix.R
+
+/**
+ * Helper for observing Translation state from browserStore.
```

Comment:
```suggestion
 * Helper for observing Translation state from [BrowserStore].
```
We should just use the class reference if the wording just works out.